omega/omega-transport/omega-transport-servicecomb/src/main/java/org/apache/servicecomb/saga/omega/transport/servicecomb/SagaConsumerHandler.java

```diff
@@ -22,30 +22,38 @@
 
 import java.lang.invoke.MethodHandles;
 
+import org.apache.servicecomb.core.Handler;
+import org.apache.servicecomb.core.Invocation;
+import org.apache.servicecomb.foundation.common.utils.BeanUtils;
 import org.apache.servicecomb.saga.omega.context.OmegaContext;
+import org.apache.servicecomb.swagger.invocation.AsyncResponse;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
 
-import org.apache.servicecomb.core.Handler;
-import org.apache.servicecomb.core.Invocation;
-import org.apache.servicecomb.swagger.invocation.AsyncResponse;
+import com.google.common.annotations.VisibleForTesting;
 
 public class SagaConsumerHandler implements Handler {
 
   private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
-  private final OmegaContext omegaContext;
 
-  public SagaConsumerHandler(@Autowired(required=false) OmegaContext omegaContext) {
-    this.omegaContext = omegaContext;
-    if (omegaContext == null) {
-      LOG.info("The OmegaContext is not injected, The SagaConsumerHandler is disabled.");
+  private OmegaContext omegaContext;
+
+  public SagaConsumerHandler() {
+    try {
+      omegaContext = BeanUtils.getBean("omegaContext");
+    } catch (NullPointerException npe) {
+      LOG.warn("The OmegaContext is not injected, The SagaConsumerHandler is disabled.");
```

Comment:
The warning message need to updated. 
SagaConsumerHandler is not disabled, it's just cannot inject transaction ID.

Reply:
Done